Sources/StreamChat/ChatClient.swift

```diff
@@ -646,7 +664,7 @@ public class ChatClient {
 
 extension ChatClient: AuthenticationRepositoryDelegate {
     func logOutUser(completion: @escaping () -> Void) {
-        logout(completion: completion)
+        logout(removeDevice: false, completion: completion)
```

Comment:
Why do we need it here?

Reply:
When logging in with a different user, we previously did not remove the device. And doing this while there is a user transition, I observed some problems, so better to remain the same behaviour as before. Either way, if this is called it means there is a bad integration, since it means the customer did not logout before login with a different user.